Makefile

```diff
@@ -55,25 +65,31 @@ check:
 		--exclude test_workloads \
 		--lib
 
+ifneq ($(VECTORS),)
+VECTOR_CARGO_FLAGS := -F vectors -Z build-std=core,alloc,proc_macro --target=riscv64gcv-unknown-none-elf.json
+else
+VECTOR_CARGO_FLAGS := 
+endif
```

Comment:
Same as above. Also, I'd just call this `CARGO_FLAGS`.

Reply:
fixed